src/utils/get-page-data.ts

```diff
@@ -46,6 +52,27 @@ export function isRuleFrontmatter(
   );
 }
 
+export function getTestAssets(dir: string, ruleBody: string): TestAssets {
+  const assetsFilenames = new Set(
+    ruleBody.match(/\/test-assets\/[^'"]*/g) ?? []
+  );
```

Comment:
I was a bit unsure about trying to parse `ruleBody` at that point rather than later when we actually build the content. But this allowed to keep the disks acces together, rather than add one in `getExamplesContent`, which seems better overall (also making tests more uniform, which is a good sign…)

Reply:
Agreed, this seems fine.